ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation       Suite to test Workload metrics feature
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource
+Resource            ../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Suite Setup         Project Suite Setup
+Suite Teardown      Project Suite Teardown
+Test Tags           DistributedWorkloadMetrics
+
+
+*** Variables ***
+${PRJ_TITLE}=    test-dw-ui
+${PRJ_TITLE_NONADMIN}=    test-dw-nonadmin
+${PRJ_DESCRIPTION}=    project used for distributed workload metrics
+
+
+*** Test Cases ***
+Verify Workload Metrics Home page Contents
+    [Documentation]    Verifies "Workload Metrics" page is accessible from
+    ...                the navigation menu on the left and page contents
+    [Tags]    Sanity
```

Comment:
Could you add some tag in the tests? RHOAIENG-4837 could be the good one (like it's done [here](https://github.com/red-hat-data-services/ods-ci/blob/4fee8773623707d9f1c6fecf291fd6663d960b1e/ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot#L34))

Reply:
sure will update the Tags , Thanks

Reply:
@FedeAlonso Can you review this Updated Tags